docker/dockerfiles/be/be_entrypoint.sh

```diff
@@ -142,16 +166,42 @@ while true; do
   #  b. coredump is enabled, and the error code is SIGABRT(6) or SIGSEGV(11);
   # otherwise BE process should still exit.
   should_exit=true
+  restart_reason=""
+
+  # Check DEBUG_MODE condition
   if [[ "$DEBUG_MODE" == "true" ]]; then
```

Comment:
1. I think you can output the error into the `be.out`. 
2. You only need to output when error. No need to output when start successfully.
3. Like DEBUG_MODE and COREDUMP_ENABLED, you can output directly in the `be.out`. Then you don't need to judge them one by one.